src/Core/Checkout/Cart/Event/BeforeLineItemAddedEvent.php

```diff
@@ -10,7 +10,7 @@
 /**
  * @deprecated tag:v6.4.0 - Will implement Shopware\Core\Framework\Event\ShopwareSalesChannelEvent
  */
-class LineItemAddedEvent extends Event /*implements ShopwareSalesChannelEvent*/
+class BeforeLineItemAddedEvent extends Event /*implements ShopwareSalesChannelEvent*/
```

Comment:
this is a break. please create a new event class and throw both events first. also add a deprecation message, for the event, which should not be dispatched anymore